Assets.php

```diff
@@ -66,6 +67,7 @@ class Assets
         'github' => 'Fxp\Composer\AssetPlugin\Repository\Vcs\GitHubDriver',
         'git-bitbucket' => 'Fxp\Composer\AssetPlugin\Repository\Vcs\GitBitbucketDriver',
         'git' => 'Fxp\Composer\AssetPlugin\Repository\Vcs\GitDriver',
+        'url' => 'Fxp\Composer\AssetPlugin\Repository\Vcs\GitDriver',
```

Comment:
To move at the end.

Reply:
Do you want to have it even under SVN?

Reply:
Moved above SVN according to the comment in code